doc/ref_model/chapters/chapter01.md

```diff
@@ -4,202 +4,321 @@
+
+<a name="1.1.3"></a>
+## 1.1.3 Common NFVI Benefits
+By providing a pre-defined environment with common capabilities, applications are able to be developed and deployed more rapidly.  In addition, the common infrastructure can be optimized for various workloads, such as IT (Information Technology), NFV, AI (Artificial Intelligence), and other future workload types as new technologies emerge. The benefits of this approach are:
+
+- Configuration automation over customization
+  - By abstracting the infrastructure capabilities as much as possible, operators are able to use common infrastructure platforms across all VNF vendors.
+  - Maintaining a consistent infrastructure allows for higher levels of automation due to a reduced need for customization of the various components.
+  - Overall, the intention is to reduce the total cost of ownership for operators and development costs for vendors
+
+- Onboarding and certification
+  - By defining abstracted infrastructure capabilities, and the metrics by which they are measured, the onboarding and certification process for both NFVI and VNFs can be standardized, potentially reducing development time for the VNF developers and deployment and operational management costs for the operators standing up the NFVI environments.
+  - Supply chain, procurement and assurance teams may then use these metrics to more accurately assess the most efficient / best value vendor for a given environment and network services requirement.
+  
+- Better utilization
+  - Properly mapping VNFs to flavours to the underlying infrastructure, brings the potential for more efficient utilization, than needing to create specific configurations for each type of application in the infrastructure.  
+  
+In conclusion, to serve the stated objective building a common NFVI infrastructure that is able to take advantage of true cloud models for the more rapid development and deployment of SDN NFV applications, the Common Telco NFVI is documentation of a reference model, a select set of architectures and a set of validation and testing suites, so that there is a more consistent model infrastructure for developers and vendors of SDN software and applications to build to.
 
 <a name="1.2"></a>
 ## 1.2 Terminology
 
-This section defines the main terms used in this document; these definitions are primarily based on the ETSI GS NFV 003 V1.4.1 (2018-08) but have been cleaned to avoid deployment technology dependencies when necessary.
+To help guide the reader, this section provides a high-level introduction to the main terms used within this document. These definitions are, with a few exceptions, based on the ETSI GS NFV 003 V1.4.1 (2018-08) definitions.  In a few cases, they have been modified to avoid deployment technology dependencies only when it seems necessary to avoid confusion.
```

Comment:
There is an empty abbreviations section that is before Chapter 1 in the RM; should we remove that abbreviations section and use this as the terminology section for the entire document?

Reply:
Agree it doesn't belong.  Will remove.

Reply:
I think we were planning to create a Glossary section at the end of the Doc in the appendix.  The idea is to keep some high level abbreviations  in Chapter 1 and move any details to the appendix.   What do others think about that idea/

Reply:
I am not sure about as an appendix, as there is some thought about putting other documents in appendices (e.g. an Intel contribution for the performance section, I believe).  The concern is that the Intel contribution isn't, as I understand, planned to be 'normative', but informative.  With terminology, I would view that as more 'normative' as it is used to define the very concepts of the model and architecture.   It would create confusion as to which appendices should be considered normative and which are informative. So while I think pulling the terminology out to a separate section is good, I don't think it should be an appendix.

Reply:
Glossary created.  Stripping down the Terminology section here.